nathanwkwong/node-ts-banking
# Db schema and migration

## What does this PR do?

- add db schema and migration

## Related Projects/Specs/Tasks (if any)?

(Include any links to related projects/specs/tasks)

## How should this be manually tested?

- run the express server and connect it to project db
- run migration script npm run migrate:genUp and verify the table created on the db

## Any background context you want to provide?(Optional)

(Add any information that will further explain or provide extra context on what drove you to create this PR)

## What are the relevant Notion URLs?

https://www.notion.so/Set-up-database-schema-and-migrations-1240975b4ef380e89fbbe96ee44d12d3?pvs=4
